### PR TITLE
Python, R: export_json bugfix, passing scale parameter to C++

### PR DESCRIPTION
**Pull request by Ilia-Shutov**
Python, R: col_sd = 1.0 by default (no scaling) as it is used in multiplying

export_json bugfix, RandomForest scale=False case

### CODE REVIEW

File: R/R/forestry.R [hunk 1]
```diff
@@ -787,7 +787,7 @@ forestry <- function(x,
 
   # Create vectors with the column means and SD's for scaling
   colMeans <- rep(0, ncol(x)+1)
-  colSd <- rep(0, ncol(x)+1)
+  colSd <- rep(1, ncol(x)+1)
```
Comment (edwardwliu) [inline R/R/forestry.R:790]: Could we leave this change out of this PR?

Comment (Ilia-Shutov) [reply R/R/forestry.R:790]: @edwardwliu Yes, but this is a fix related to R - colSd is 1 by default - same fix for Python is `col_sd = np.repeat(1.0, ncol + 1)`, other code in this PR is about code style update + added test.
Do you want me to fix only Python part for now?

Comment (Ilia-Shutov) [reply R/R/forestry.R]: Fixed by reverting colSd back to 0 init and passing scale bool parameter to C++ function

**Review by edwardwliu — APPROVED** [review]
LGTM

**Pull request merged**